app/views/notes/_table.html.slim

```diff
@@ -0,0 +1,50 @@
+table.datagrid.striped.dataTable.no-footer
```

Comment:
This is the pattern for the tables I think we can follow for filterable tables. Do we need a usage docs for this @benwbrum ?